modules/ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces_{context}"]
+= Creating the default revision tag and relabeling the namespaces
+
+This procedure explains how to create the default revision tag and relabel the namespaces after you have completed the {SMProduct} 2 to {SMProduct} 3 cluster-wide migration process by using the {istio} injection label.
+
+The `bookinfo` example application is being used for demonstration purposes.
+
+.Prerequisites
+
+* You have completed the {SMProduct} 2 to {SMProduct} 3 cluster-wide migration process by using {istio} injection label.
+
+.Procedure
+
+. Create a YAML files called `rev-tag.yml` that defines the `IstioRevisionTag` resource:
```

Comment:
```suggestion
. Create a YAML file called `rev-tag.yaml` that defines the `IstioRevisionTag` resource:
```

Replace/yml/yaml

Also the `oc apply` command below uses `.yaml`

Reply:
Updated files to file. Thanks for catching this. In this doc set, we typically use the .yml extension. Both the .yml and .yaml extensions are used in industry. The developers said they see .yml more frequently.

Reply:
Updated. Done.